client/lib/analytics/index.js

```diff
@@ -266,11 +277,13 @@ const analytics = {
 
 				const json = JSON.stringify( {
 					beacons: [
-						'calypso.' + config( 'boom_analytics_key' ) + '.' + featureSlug + '.' + eventType.replace( '-', '_' ) + ':' + duration + '|ms'
+						'calypso.' + config( 'boom_analytics_key' ) + '.' + featureSlug + '.' +
+							eventType.replace( '-', '_' ) + ':' + duration + '|ms'
 					]
 				} );
 
-				new Image().src = 'https://pixel.wp.com/boom.gif?v=calypso&u=' + encodeURIComponent( pageUrl ) + '&json=' + encodeURIComponent( json );
+				new Image().src = 'https://pixel.wp.com/boom.gif?v=calypso&u=' + encodeURIComponent( pageUrl ) +
+					'&json=' + encodeURIComponent( json );
```

Comment:
these strings are getting long as well and odd to read. could we clean up with some variables/template literals?

```js
const type = eventType.replace( '-', '_' );
`calypso.${ config( 'boom_analytics_key' ) }.${ featureSlug }.${ type }:${ duration }|ms`

const [ url, jsonData ] = [ pageUrl, json ].map( encodeURIComponent );
`https://pixel.wp.com/boom.gif?v=calypso&u=${ url }&json=${ jsonData }`
```